linux/datadog_wrapper

```diff
@@ -1,62 +1,79 @@
 #!/usr/bin/env sh
 
-CURRENT_DIR=$(pwd)
+main() {
+    # Required to add the AAS metadata
+    export DD_AZURE_APP_SERVICES=1
 
-# Required to add the AAS metadata 
-export DD_AZURE_APP_SERVICES=1
+    echo "Setting application directory as $APP_DIR"
+    APP_DIR=$(pwd)
+   
+    echo "Setting Datadog environment variables"
+    setEnvVars
 
-if [ -z "$DD_DIR" ]; then
-    DD_DIR="/home/datadog"
-fi
+    echo "Creating and switching to the Datadog directory at $DD_DIR"
+    mkdir -p $DD_DIR && cd $DD_DIR || return
 
-if [ -z "$DD_BINARIES_URL" ]; then
-    DD_BINARIES_URL="https://github.com/DataDog/datadog-aas-extension/releases/download"
-fi
+    if [ "$WEBSITE_STACK" = "NODE" ]; then
```

Comment:
This might be a good use for bash's `case` statements. Example from internet:

```bash
case EXPRESSION in

  PATTERN_1)
    STATEMENTS
    ;;

  PATTERN_2)
    STATEMENTS
    ;;

  PATTERN_N)
    STATEMENTS
    ;;

  *)
    STATEMENTS
    ;;
esac
```